src/pages/Greeting.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { useFetch } from '../hooks/useFetch';
+import {
+  fetchRandomMessageRequest,
+} from '../store/messageSlice';
+
+export const Greeting = () => {
+  const url = 'http://127.0.0.1:4000/fetch_all_messages';
+  const { loading, error, randomMessage } = useFetch({ url });
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchRandomMessageRequest());
+  }, [dispatch]);
+
+  if (loading) {
+    return <div>Loading...</div>;
+  }
+
+  if (error) {
+    return (
+      <div>
+        Error:
+        {error}
+      </div>
+    );
+  }
+
+  return (
+    <div>
+      <h1>Random Message:</h1>
+      {randomMessage && <p>{randomMessage.content}</p>}
+    </div>
+  );
+};
```

Comment:
- [ ] I didn't get a random message on the screen when I ran the dev server. Please fix that.

![image](https://github.com/kiboma2021/hello-react-front-end/assets/68796232/e1ce14b1-6d5a-4937-a12e-31214ff475cc)

Reply:
This has now been fixed

Reply:
![Screenshot from 2024-02-15 16-37-01](https://github.com/kiboma2021/hello-react-front-end/assets/93031593/7af9fb31-86bd-45b7-870a-bcf66a3ef395)